src/components/ForecastDetails.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/ForecastDetails.css";
+
+function ForecastDetails({ forecast }) {
+  const { date, temperature, wind, humidity } = forecast;
+  const formattedDate = new Date(date).toDateString();
+
+  return (
+    <div className="forecast-details" data-testid="forecast-details">
+      <div className="forecast-details__date">Date: {formattedDate}</div>
+      <div className="forecast-details__maxTemperature">
```

Comment:
CSS classes are almost always written in kebab case with all lowercase letters. In this case this one should be `forecast-details__max-temperature`

That said, the only selector you have defined properties in the styles for is `forecast-details` itself, were you planning on adding more styles for these other elements?